app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split = command.split(" ")
```

Comment:
split is not clear name
also you must validate before that command.split() has exactly 3 elements